cpu/nrf5x_common/include/periph_cpu_common.h

```diff
@@ -143,6 +153,7 @@ typedef struct {
  * @brief   Override SPI mode values
  * @{
  */
+#ifndef CPU_FAM_NRF9160
```

Comment:
Is this necessary?

Reply:
Yes, at least for some time. Otherwise we get some compilations errors.
I'll fix it properly in a followup when I'll add SPI support.